src/test/rustdoc/issue-43869.rs

```diff
@@ -2,6 +2,7 @@ pub fn g() -> impl Iterator<Item=u8> {
     Some(1u8).into_iter()
 }
 
+#[allow(unused_parens)]
```

Comment:
Why is this allowed? It looks like this test ..
 Doesn't test anything?

Reply:
By test, I assume you mean the `h` function? If so, then yes; I don't think it does anything that `g` doesn't already test. But I wasn't sure if there was a reason for it, and I wanted to keep this change small if possible.

Reply:
Oh, all the `@has` annotations are at the bottom of the file for some reason. That would be nice to fix but it doesn't have to be here.